source/_components/alarm_control_panel.lupusec.markdown

```diff
@@ -0,0 +1,36 @@
+---
+layout: page
+title: "Lupusec Alarm Control Panel"
+description: "Instructions on how to setup the Lupusec Alarm control panel within Home Assistant."
+date: 2018-10-03 16:00
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: lupusec.jpg
+ha_category: Alarm
+ha_release: 0.79
+ha_iot_class: "Cloud Push"
+---
+
+The `lupus electronics` security control panel platform allows you to control your [Lupusec](https://www.lupus-electronics.de) devices.
+
+The requirement is that you have setup your [Lupusec component](/components/lupusec/).
+
+To use Lupusec devices in your installation, add the following `lupusec` section to your `configuration.yaml` file:
+
+```yaml
+# Example configuration.yaml entry
+lupusec:
+      username: user 
+      password: 'greatpassword'
+      ip_address: 192.168.178.35
+      scan_interval: 2
```

Comment:
We try to keep all configuration samples minimal. Thus, no optional requirement in the default sample. This helps a user to get started quickly by copy-&-paste the sample without worrying about optional parameters which they most likely not need. If required, insert a full configuration sample later that covers special setups or alike.